dolphinscheduler-bom/pom.xml

```diff
@@ -748,6 +749,102 @@
                 <artifactId>snakeyaml</artifactId>
                 <version>${snakeyaml.version}</version>
             </dependency>
+            <!-- https://mvnrepository.com/artifact/software.amazon.awssdk/datasync -->
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>datasync</artifactId>
+                <version>${datasync.version}</version>
+               <!-- <exclusions>
```

Comment:
remove if no need.

Reply:
This comment is not addressed

Reply:
sorry, the page not refreshed and the comment not show when i merge this. have fixed this in this pr.
https://github.com/apache/dolphinscheduler/pull/12273